pkgs/desktops/plasma-5/kwin/default.nix

```diff
@@ -21,22 +21,20 @@ mkDerivation {
     epoxy libICE libSM libinput libxkbcommon udev wayland xcb-util-cursor
     xwayland
 
-    qtdeclarative qtmultimedia qtscript qtx11extras
+    qtdeclarative qtmultimedia qtscript qtx11extras qtsensors
 
     breeze-qt5 kactivities kcmutils kcompletion kconfig kconfigwidgets
     kcoreaddons kcrash kdeclarative kdecoration kglobalaccel ki18n kiconthemes
     kidletime kinit kio knewstuff knotifications kpackage kscreenlocker kservice
     kwayland kwidgetsaddons kwindowsystem kxmlgui plasma-framework
   ];
   outputs = [ "bin" "dev" "out" ];
-  patches = copyPathsToStore (lib.readPathsFromFile ./. ./series)
-    ++ [(fetchpatch {
-        name = "cmake-3.10.diff";
-        # included upstream for kwin >= 5.11.95
-        url = "https://github.com/KDE/kwin/commit/cd544890ced4192.diff";
-        sha256 = "0z5nbcg712v10mskb7r9v0jcx5h8q4ixb7fjbb0kicmzsc266yd5";
-    })]
-    ;
+  patches = copyPathsToStore (lib.readPathsFromFile ./. ./series) ++ [
+    (fetchpatch {
+      url = "https://phabricator.kde.org/file/data/ojbvgljuhqxztrdcicoy/PHID-FILE-6wvjkli2blrgw7jei3qg/D26720.diff";
```

Comment:
The url does not work for me.
The following works for me:
https://phabricator.kde.org/file/data/5bu6h2c2ksaqctyy275b/PHID-FILE-jtbgqz2heluxsovlqyv3/D10373.diff

Suspicious, that your url also has a different filename?

Reply:
Strange. That was working at some point.

How stable are phabricator urls?

Reply:
I don't know :(

Reply:
@adisbladis if you want to be on the save side, the patch got merged: https://github.com/KDE/kwin/commit/6e5f5d92daab4c60f7bf241d90a91b3bea27acfd

Reply:
Perfect! I'll use the github link.